WordPress/src/main/java/org/wordpress/android/ui/reader/tracker/ReaderTracker.kt

```diff
@@ -465,6 +466,23 @@ class ReaderTracker @Inject constructor(
             AnalyticsTracker.track(stat, properties)
         }
 
+        private fun trackFollowedCount(type: String, numberOfItems: Int) {
+            val props: MutableMap<String, String> = HashMap()
+            props["type"] = type
+            props["count"] = numberOfItems.toString()
+            AnalyticsTracker.track(Stat.READER_FOLLOWING_FETCHED, props)
+        }
+
+        @JvmStatic
```

Comment:
If the idea of moving it to the `ViewModel` makes sense, maybe we can remove the `@JvmStatic` annotation from here and line `481`.

Reply:
Hey @RenanLukas Thank you for your review! Your suggestions are valuable, and I will work on the PR to enhance site tracking and transfer the logic to the ViewModel within that PR.